### PR TITLE
Add USDTETHV3-AIP

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: content/aips/USDTETHV3-AIP.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+title: USDTethV3-AIP
```
Comment (sakulstra) [inline content/aips/USDTETHV3-AIP.md:2]: Better have a descriptive title as this is what will be shown on the governance UIs
```suggestion
Add USDT to Aave Ethereum V3 pool
```

File: content/aips/USDTETHV3-AIP.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+The Interest rate Strategy curve is the same as for USDT in V2. for now.
+The ACI is supportive of onboarding USDT as it was in V2.
+For now, the ACI doesn’t consider V3 mature enough to activate a USD stablecoins emode and consider USDT in it.
+Also, supply and borrow caps are suggested for this asset to limit risk exposure.
+
+## Specification
+
+Ticker: USDT
+
+Contract Address: 0xdAC17F958D2ee523a2206206994597C13D831ec7
```
Comment (sakulstra) [inline content/aips/USDTETHV3-AIP.md:29]:
```suggestion
Contract Address: [0xdAC17F958D2ee523a2206206994597C13D831ec7](https://etherscan.io/address/0xdAC17F958D2ee523a2206206994597C13D831ec7)
```

File: content/aips/USDTETHV3-AIP.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+The proposal Payload was reviewed by Bored Ghost Developpers.
+
+## Deployed Contracts
+
+- [AaveV3EthUSDTPayload: ProposalPayload](https://etherscan.io/address/FILL_THIS#code)
+
+## Copyright
+
+Copyright and related rights waived via CC0.
```
Comment (sakulstra) [inline content/aips/USDTETHV3-AIP.md:92]:
```suggestion
Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```
for consistency's sake with other proposals

File: content/aips/USDTETHV3-AIP.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+A list of relevant links like for this proposal e.g.
+
+- [forum discussion]([forumlink](https://governance.aave.com/t/arfc-add-usdt-to-ethereum-v3-market/11536))
+- [tests](githublink)("")
+- [proposalCode](githublink)("")
+
+## Security Considerations
+
+The proposal Payload was reviewed by Bored Ghost Developpers.
```
Comment (sakulstra) [inline content/aips/USDTETHV3-AIP.md:84]:
```suggestion
The proposal Payload was reviewed by [Bored Ghost Developing](https://bgdlabs.com/).
```

File: content/aips/USDTETHV3-AIP.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+|Enable Borrow|YES|
+|Enable Collateral|NO|
+|Loan To Value|N/A|
+|Liquidation Threshold|N/A|
+|Liquidation Bonus|N/A|
+|Reserve Factor|10%|
+|Liquidation Protocol Fee|0.1|
+|Borrow Cap|185M|
+|Supply Cap|200M|
+|Debt Ceiling|185M|
```
Comment (sendra) [inline content/aips/USDTETHV3-AIP.md:43]: This should be N/A as is configured on the Payload, as USDT will not be added to Isolation mode

File: content/aips/USDTETHV3.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+      eModeCategory: 0
+    });
+```
+
+## References
+
+A list of relevant links like for this proposal e.g.
+
+- [forum discussion]([forumlink](https://governance.aave.com/t/arfc-add-usdt-to-ethereum-v3-market/11536))
+- [tests](githublink)("https://github.com/bgd-labs/aave-v3-crosschain-listing-template/blob/master/src/test/mainnet/AaveV3USDTETHPayloadTest.t.sol")
```
Comment (sendra) [inline content/aips/USDTETHV3.md:79]:
```suggestion
- [tests](https://github.com/bgd-labs/aave-v3-crosschain-listing-template/blob/master/src/test/mainnet/AaveV3USDTETHPayloadTest.t.sol)
```

File: content/aips/USDTETHV3.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+      liqProtocolFee: 10_00,
+      eModeCategory: 0
+    });
+```
+
+## References
+
+A list of relevant links like for this proposal e.g.
+
+- [forum discussion]([forumlink](https://governance.aave.com/t/arfc-add-usdt-to-ethereum-v3-market/11536))
```
Comment (sendra) [inline content/aips/USDTETHV3.md:78]:
```suggestion
- [forum discussion](https://governance.aave.com/t/arfc-add-usdt-to-ethereum-v3-market/11536)
```

File: content/aips/USDTETHV3.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+- [tests](githublink)("https://github.com/bgd-labs/aave-v3-crosschain-listing-template/blob/master/src/test/mainnet/AaveV3USDTETHPayloadTest.t.sol")
+- [proposalCode](githublink)("https://github.com/bgd-labs/aave-v3-crosschain-listing-template/blob/master/src/contracts/mainnet/AaveV3EthUSDTPayload.sol")
```
Comment (sendra) [inline content/aips/USDTETHV3.md:80]:
```suggestion
- [proposalCode](https://github.com/bgd-labs/aave-v3-crosschain-listing-template/blob/master/src/contracts/mainnet/AaveV3EthUSDTPayload.sol)
```

**Review by sendra — APPROVED** [review]

File: content/aips/USDTETHV3.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+discussions: https://governance.aave.com/t/arfc-add-usdt-to-ethereum-v3-market/11536
+created: 2023-02-07
+updated: 2023-02-07
+---
+
+## Simple Summary
+This AIP proposal presents Aave with the opportunity to onboard USDT to the Ethereum v3 Liquidity Pool.
+
+## Motivation
+Supporting stablecoin diversity is part of the Aave-Chan Initiative (ACI) delegate platform.cd ..
```
Comment (sakulstra) [inline content/aips/USDTETHV3.md:14]:
```suggestion
Supporting stablecoin diversity is part of the Aave-Chan Initiative (ACI) delegate platform.
```

**Review by sakulstra — APPROVED** [review]

**Pull request merged**